src/current/cockroachcloud/plan-your-cluster.md

```diff
@@ -18,15 +18,15 @@ Before making any changes to your cluster's configuration, review the requiremen
 
 A single node cluster is only appropriate for single-region application development and testing. For single-region production deployments, we recommend a minimum of 3 nodes. The number of nodes you choose also affects your storage capacity and performance. See the [Example](#dedicated-example) for more information.
 
-All {{ site.data.products.db }} clusters use 3 Availability Zones (AZs). For balanced data distribution and best performance, we recommend using a number of nodes that is a multiple of 3 (e.g., 3, 6, or 9 nodes per region).
+All {{ site.data.products.dedicated }} clusters use 3 Availability Zones (AZs). For balanced data distribution and best performance, we recommend using a number of nodes that is a multiple of 3 (e.g., 3, 6, or 9 nodes per region).
```

Comment:
```suggestion
Each {{ site.data.products.dedicated }} cluster uses three Availability Zones (AZs). For balanced data distribution and best performance, we recommend using a number of nodes that is a multiple of 3 (for example, 3, 6, or 9 nodes per region).
```
In general, words for numerals less than 10. I left the ones at the end alone because we are doing math at that point. In general, I think it's more clear to talk about things like clusters and organizations in the singular. WDYT?